packages/airnode-examples/README.md

```diff
@@ -143,9 +146,9 @@ step. The latter, `secrets.env` can be generated it using:
 yarn create-airnode-secrets
 ```
 
-> If you are not using docker for linux and you want to connect to your local hardhat network, you will need to modify
-> the generated `secrets.env` file found in `integrations/<integration-name>/` by replacing the provider URL with the
-> following: `PROVIDER_URL=http://host.docker.internal:8545`. This is a docker limitation. See:
+> If you are not using docker for Windows/WSL or docker for mac and you want to connect to your local hardhat network,
```

Comment:
I believe "not" should be removed based both on your comment and the fact that I have to do this for WSL myself
```suggestion
> If you are using docker for Windows/WSL or docker for mac and you want to connect to your local hardhat network,
```

Reply:
Yeah, thanks.